app/controllers/integrations/base_controller.rb

```diff
@@ -2,11 +2,18 @@ class Integrations::BaseController < ApplicationController
   skip_around_action :login_user
   skip_before_action :verify_authenticity_token
   before_action :validate_request
+  after_action :record_webhook
 
   def create
-    return head(:ok) unless deploy?
+    unless deploy?
+      record_log :error, "Request is not a deploy"
```

Comment:
Not really an error. We'll probably get notifications for a bunch of stuff that shouldn't be a deploy, like a PR against a branch that isn't master.